### PR TITLE
Added logout executable data with test-case

### PR DESCRIPTION
**Pull request by gulshanvasnani**
PR contains :-
1. Executable data for logging out session keys.
2. Positive test case to verify the event fired on logging out all the sessions.

Fixes :- #75

### CODE REVIEW

File: lib/helper/TokenHolder.js [hunk 1]
```diff
@@ -79,6 +79,21 @@ class TokenHolder {
     return executableData;
   }
 
+  /**
+   * Logout all authorized sessions.
+   *
+   * @returns {*} Executable data to logout sessions.
+   */
+  getLogoutExecutableData() {
```
Comment (abhayks1) [inline lib/helper/TokenHolder.js]: We should add rawTx method.

File: lib/helper/TokenHolder.js [hunk 2]
```diff
@@ -79,6 +79,21 @@ class TokenHolder {
...
+    const oThis = this;
+
+    const jsonInterface = oThis.abiBinProvider.getABI(tokenHolderContractName),
+      contract = new oThis.auxiliaryWeb3.eth.Contract(jsonInterface, oThis.tokenHolderProxy),
+      executableData = contract.methods.logout().encodeABI();
```
Comment (abhayks1) [inline lib/helper/TokenHolder.js]: Let's refactor below two lines in a single method and call it from different methods in this file:
```
const jsonInterface = oThis.abiBinProvider.getABI(tokenHolderContractName),
      contract = new oThis.auxiliaryWeb3.eth.Contract(jsonInterface, oThis.tokenHolderProxy),
```
It's used in multiple methods.

Comment (gulshanvasnani) [reply lib/helper/TokenHolder.js]: `getTokenHolderInstance` method added.

**Comment by abhayks1** [comment]
@pgev @benjaminbollen Need your inputs also in the PR.

**Pull request merged**